simulatingrisk/hawkdovevar/app.py

```diff
@@ -10,8 +10,30 @@
 )
 from simulatingrisk.hawkdove.app import plot_hawks
 
+# adjust single-risk params for variable risk
 jupyterviz_params_var = jupyterviz_params.copy()
+# remove parameter for agent risk level;
```

Comment:
Why is this removed? not clear from immediate context but perhaps clear elsewhere